Assets/js/global-revenue.js

```diff
@@ -1,112 +1,121 @@
 mQuery(document).ready(function () {
-    mQuery.ajax({
-        url: mauticAjaxUrl,
-        type: 'POST',
-        data: {
-            action: 'plugin:mauticContactLedger:globalRevenue',
-        },
-        cache: true,
-        dataType: 'json',
-        success: function (response) {
-
-            var rowCount = Math.floor((widgetHeight - 220) / 40);
-            mQuery('#global-revenue').DataTable({
-                data: response.rows,
-                autoFill: true,
-                columns: response.columns,
-                order: [[3, 'desc']],
-                bLengthChange: false,
-                lengthMenu: [[rowCount]],
-                columnDefs: [
-                    {
-                        render: function (data, type, row) {
-                            return renderPublishToggle(row[1], row[0]);
-                        },
-                        targets: 0
-                    },
-                    {
-                        render: function (data, type, row) {
-                            return renderCampaignName(row);
-                        },
-                        targets: 2
-                    },
-                    {
-                        render: function (data, type, row) {
-                            return '$' + data;
-                        },
-                        targets: [5, 6, 7, 9]
-                    },
-                    {
-                        render: function (data, type, row) {
-                            return data + '%';
-                        },
-                        targets: 8
+    mQuery.getScriptCachedOnce('/plugins/MauticContactLedgerBundle/Assets/js/datatables.min.js', function () {
```

Comment:
This PR looks good!

Instead of using relative pathing (assumes we are always running from the root domain space without path sharing), we should be using the Mautic global variables mauticBasePath and mauticAssetPrefix.

Instead of `mQuery.getScriptCachedOnce('/plugins`
Use `mQuery.getScriptCachedOnce(mauticBasePath + '/' + mauticAssetPrefix + 'plugins`

Reply:
fixed